### DISCUSSION TITLE
Latest github release has no binaries attached

### DISCUSSION THREAD

**Discussion by kristianheljas** [arendst/Tasmota#12060] [Q&A] [answered]
Hey,

Latest [9.4.0 Leslie](https://github.com/arendst/Tasmota/releases/tag/v9.4.0) release in GitHub has no binaries attached, all previous ones do.

@arendst Was this intentional or simply forgotten?

For example, this breaks [TasmoAdmin](https://github.com/reloxx13/TasmoAdmin) automatic firmware update functionality which relies on [Github releases API](https://api.github.com/repos/arendst/Tasmota/releases/latest) to fetch latest binary.

One would create PR for TasmoAdmin to use http://ota.tasmota.com/tasmota/release/ instead, just hoping for an easy fix here :)

**Comment by arendst** [comment, marked as answer]
Intentional.

The easy fix should be done in TasmoAdmin. See https://github.com/reloxx13/TasmoAdmin/issues/427#issuecomment-826280418

**Reply by kristianheljas** [reply]
Thanks for clarification and the link I wasn't able to find! :)

**Reply by sfromis** [reply]
FTR, TasmoAdmin is known to hammer devices so hard that some may become unresponsive. We usually recommend other tools instead, like [Tasmota Device Manager](https://github.com/jziolkowski/tdm#tasmota-device-manager).
